src/Persistence/MassTransit.Azure.Table/Saga/PartitionSagaKeyFormatter.cs

```diff
@@ -0,0 +1,23 @@
+namespace MassTransit.Azure.Table.Saga
+{
+    using System;
+    using MassTransit.Saga;
+
+
+    public class PartitionSagaKeyFormatter<TSaga> :
+        ISagaKeyFormatter<TSaga>
+        where TSaga : class, ISaga
+    {
+        readonly string _rowKey;
+
+        public PartitionSagaKeyFormatter(string rowKey)
```

Comment:
Naming here is throwing me off - might just be a PR thing. ParitionSagaKeyFormatter but row key as the parameter feels off 🤔

Reply:
Yeah, naming is hard I'm open to suggestions :)

Reply:
renamed.